frontend/src/utils/toasts/errorToast.js

```diff
@@ -0,0 +1,23 @@
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
I'm not sure, but you can only import this styling once in the component where you used <ToastContainer />